### PR TITLE
Probability plot and table- Process capability Updates 

### PR DESCRIPTION
**Pull request by tomishay**
Updates:
- Lognormal distribution 
- Weibull distributon 
- Table

### CODE REVIEW

**Comment by koenderks** [comment]
If you have a common function like `ProbabilityPlotNoId`, can you adjust it in the common file instead of adding it to the process capability analysis?

**Comment by tomishay** [comment]
I just adjusted the Common R file and the rest of the sections so there is no overlap between the functions.

**Comment by koenderks** [comment]
@tomishay It has nothing to do with your code

File: R/processCapability.R [hunk 1]
```diff
@@ -16,35 +16,57 @@
...
   subgroupsName <- options$subgroups
   makeSubgroups <- subgroupsName != ""
-  
+
   if (is.null(dataset)) {
     if (makeSubgroups) {
-      dataset         <- .readDataSetToEnd(columns.as.numeric = diameter, columns.as.factor = subgroupsName)
-      dataset.factors <- .readDataSetToEnd(columns = diameter, columns.as.factor = subgroupsName)
+      dataset         <- na.omit(.readDataSetToEnd(columns.as.numeric = variables, columns.as.factor = subgroupsName))
+      dataset.factors <- na.omit(.readDataSetToEnd(columns = variables, columns.as.factor = subgroupsName))
```
Comment (vandenman) [inline R/processCapability.R:27]: there's an argument for this already
```suggestion
      dataset         <- .readDataSetToEnd(columns.as.numeric = variables, columns.as.factor = subgroupsName, exclude.na.listwise = c(variables, subgroupsName))
      dataset.factors <- .readDataSetToEnd(columns = variables, columns.as.factor = subgroupsName, exclude.na.listwise = c(variables, subgroupsName))
```

File: inst/qml/processCapability.qml [hunk 2]
```diff
@@ -18,90 +17,90 @@ import QtQuick.Layouts 							1.3
...
-		CheckBox { name: "initialProbabilityPlot";		label: qsTr("Normal Probability Plot")			}
-		CheckBox { name: "initialCapabilityAnalysis";	label: qsTr("Process Capability of Diameter")	}
-	}
+  usesJaspResults:							true
+  columns:									1
 
-	Section
-	{
-		title: qsTr("Follow-up Process Capability Study")
+  VariablesForm
```
Comment (vandenman) [inline inst/qml/processCapability.qml:25]: please indent qml with tabs, not spaces.

**Review by vandenman — CHANGES_REQUESTED** [review]
looks good! two small comments below

**Pull request merged**